### PR TITLE
improve descriptions for searching for federation

### PR DESCRIPTION
**Pull request by xundeenergie**
As discussed in matrix, i wrote an improvement for the docu for searching for communities, posts and comments.

### CODE REVIEW

File: src/en/federation/federation.md [hunk 1]
```diff
@@ -13,4 +13,31 @@ One way you can take advantage of federation is by opening a different instance,
...
+## Searching for communities
+
+If you search for a community first time, 20 posts are fetched initially. Only if a least one user on your instance subscibes this community on the other host, this host will deliver updates to your instance.
+Updates mean: 
+- new posts
+- new comments
+- votings
+- edited comments and posts
+
+You can copy the URL of the community from the address-bar in your browser and insert it in your search-field. Wait a few seconds, the post will appear below. There is no indicator for a running search now!
```
Comment (Nutomic) [inline src/en/federation/federation.md:27]: For comments, the format is `https://lemmy.ml/comment/456`, so you cant copy it directly and have to remove  the `/post/123` part first.

File: src/en/federation/federation.md [hunk 2]
```diff
@@ -13,4 +13,31 @@ One way you can take advantage of federation is by opening a different instance,
...
+You can see the list of linked instances by following the "Instances" link at the bottom of any Lemmy page.
+
+## Searching for communities
+
+If you search for a community first time, 20 posts are fetched initially. Only if a least one user on your instance subscibes this community on the other host, this host will deliver updates to your instance.
+Updates mean: 
+- new posts
+- new comments
+- votings
+- edited comments and posts
```
Comment (Nutomic) [inline src/en/federation/federation.md:25]: This is not a complete list, but its enough for here. Maybe add "mod actions" as a general category.

File: src/en/federation/federation.md [hunk 3]
```diff
@@ -13,4 +13,31 @@ One way you can take advantage of federation is by opening a different instance,
 - `https://lemmy.ml/post/123` (Post)
 - `https://lemmy.ml/comment/321` (Comment)
 
-You can see the list of linked instances by following the "Instances" link at the bottom of any Lemmy page.
\ No newline at end of file
+You can see the list of linked instances by following the "Instances" link at the bottom of any Lemmy page.
+
+## Searching for communities
```
Comment (Nutomic) [inline src/en/federation/federation.md:18]: Searching isnt really the right term. "Fetching" is more accurate.

File: src/en/federation/federation.md [hunk 4]
```diff
@@ -13,4 +13,31 @@ One way you can take advantage of federation is by opening a different instance,
...
+Copy the URL of a post in your lemmy's search-field. Wait a few seconds until the post will appear. Klick on it, you'll be lead to the posting into the community, where you cann see the "Follow"-button and follow the community. Be aware, not all postings are fetched from the community, only a few ammount, and no comments!
+
+## Searching for comments
+
+If you find an interesting comment under a posting on another instance, you can find below the comment in the 3-dot-menu the link-symbol. Copy this link, oder klick on the link-button and copy the url from your address-bar in the browser. The link looks like `https://lemmy.ml/post/56382/comment/40796`. Remove the `post/XXX` part from this link and put it into your search-bar. For this example, search for `https://lemmy.ml/comment/40796`. 
+This comment, all parent comments, users and community and the responding posting is fetched from the remote instance, if they are not known locally.
+No siblings are fetched!!
+
+If you wand more comments from older postings, you have to search for them the same way as described just above.
+
```
Comment (Nutomic) [inline src/en/federation/federation.md:40]: Users can also be fetched by URL, but that currently doesnt fetch any further data.

File: src/en/federation/federation.md [hunk 5]
```diff
@@ -13,4 +13,31 @@ One way you can take advantage of federation is by opening a different instance,
...
+- new posts
+- new comments
+- votings
+- edited comments and posts
+
+You can copy the URL of the community from the address-bar in your browser and insert it in your search-field. Wait a few seconds, the post will appear below. There is no indicator for a running search now!
+
+## Searching for posts
+
+Copy the URL of a post in your lemmy's search-field. Wait a few seconds until the post will appear. Klick on it, you'll be lead to the posting into the community, where you cann see the "Follow"-button and follow the community. Be aware, not all postings are fetched from the community, only a few ammount, and no comments!
```
Comment (Nutomic) [inline src/en/federation/federation.md:31]: That part about the follow button is confusing, dont think it belongs here.

File: src/en/federation/federation.md [hunk 6]
```diff
@@ -13,4 +13,31 @@ One way you can take advantage of federation is by opening a different instance,
...
+
+If you find an interesting comment under a posting on another instance, you can find below the comment in the 3-dot-menu the link-symbol. Copy this link, oder klick on the link-button and copy the url from your address-bar in the browser. The link looks like `https://lemmy.ml/post/56382/comment/40796`. Remove the `post/XXX` part from this link and put it into your search-bar. For this example, search for `https://lemmy.ml/comment/40796`. 
+This comment, all parent comments, users and community and the responding posting is fetched from the remote instance, if they are not known locally.
+No siblings are fetched!!
+
+If you wand more comments from older postings, you have to search for them the same way as described just above.
+
+## Drawbacks
+
+There is no possibility to fetch easily old postings or comments from a remote instance from your instance, if you have joined a community.
```
Comment (Nutomic) [inline src/en/federation/federation.md:43]: Calling it "limitation" would sound a bit nicer ;)

File: src/en/federation/federation.md [hunk 7]
```diff
@@ -1,5 +1,5 @@
 # Federation
-
+https://lemmy.ml/u/20776
```
Comment (dessalines) [inline src/en/federation/federation.md:2]: Whats this line about also.

File: src/en/federation/federation.md [hunk 8]
```diff
@@ -13,4 +13,31 @@ One way you can take advantage of federation is by opening a different instance,
...
 - `https://lemmy.ml/comment/321` (Comment)
 
-You can see the list of linked instances by following the "Instances" link at the bottom of any Lemmy page.
\ No newline at end of file
+You can see the list of linked instances by following the "Instances" link at the bottom of any Lemmy page.
+
+## Searching for communities
+
+If you search for a community first time, 20 posts are fetched initially. Only if a least one user on your instance subscibes this community on the other host, this host will deliver updates to your instance.
+Updates mean:
```
Comment (dessalines) [inline src/en/federation/federation.md:21]: Always use 2 newlines for new paragraphs for lines.

File: src/en/federation/federation.md [hunk 9]
```diff
@@ -13,4 +13,31 @@ One way you can take advantage of federation is by opening a different instance,
...
 
-You can see the list of linked instances by following the "Instances" link at the bottom of any Lemmy page.
\ No newline at end of file
+You can see the list of linked instances by following the "Instances" link at the bottom of any Lemmy page.
+
+## Searching for communities
+
+If you search for a community first time, 20 posts are fetched initially. Only if a least one user on your instance subscibes this community on the other host, this host will deliver updates to your instance.
+Updates mean: 
+- new posts
```
Comment (dessalines) [inline src/en/federation/federation.md:22]: Always add 2 newlines before lists.

File: src/en/federation/federation.md [hunk 10]
```diff
@@ -13,4 +13,31 @@ One way you can take advantage of federation is by opening a different instance,
...
+You can copy the URL of the community from the address-bar in your browser and insert it in your search-field. Wait a few seconds, the post will appear below. There is no indicator for a running search now!
+
+## Searching for posts
+
+Copy the URL of a post in your lemmy's search-field. Wait a few seconds until the post will appear. Klick on it, you'll be lead to the posting into the community, where you cann see the "Follow"-button and follow the community. Be aware, not all postings are fetched from the community, only a few ammount, and no comments!
+
+## Searching for comments
+
+If you find an interesting comment under a posting on another instance, you can find below the comment in the 3-dot-menu the link-symbol. Copy this link, oder klick on the link-button and copy the url from your address-bar in the browser. The link looks like `https://lemmy.ml/post/56382/comment/40796`. Remove the `post/XXX` part from this link and put it into your search-bar. For this example, search for `https://lemmy.ml/comment/40796`. 
+This comment, all parent comments, users and community and the responding posting is fetched from the remote instance, if they are not known locally.
```
Comment (dessalines) [inline src/en/federation/federation.md:36]: Same newline stuff as above.

**Review by dessalines — CHANGES_REQUESTED** [review]

**Comment by Nutomic** [comment]
@xundeenergie Are you still planning to update this PR? If not, I can make the necessary changes.

**Comment by Nutomic** [comment]
I'm finishing this in https://github.com/LemmyNet/lemmy-docs/pull/46.

**Pull request closed without merging**